app/assets/stylesheets/theme.scss

```diff
@@ -17,5 +18,9 @@ $input-btn-focus-width: .2rem;
 $color-hover: $green;
 $active-background: white;
 $active-border: $teal;
+$btnselfborder: #2f8875;
```

Comment:
`button-border-color` ?

Reply:
Yes, when seeing projects of self.

Reply:
I think, he means that maybe you can use a little better naming convention.